redaxo/src/core/pages/packages.php

```diff
@@ -6,6 +6,40 @@
 
 $subpage = rex_request('subpage', 'string');
 
+function packages_title() {
```

Comment:
da wir hier eine spezielle pages struktur haben, habe ich das rendering der neuen tab-navi in diese funktion verlagert.

diese wird in allen pages verwendet.